src/cfn.py

```diff
@@ -202,16 +202,10 @@ def cmd(stackname, command=None, username=DEPLOY_USER):
     print "Connecting to: %s" % stackname
     stack_all_ec2_nodes(
         stackname,
-        cmd_parallel_subtask,
+        (parallel(lambda command: run(command)), {'command': command}),
```

Comment:
that's what I was looking for, thanks: a more idiomatic way to do this in Fabric that didn't require all the bureaucracy of separate tasks